feat: set context during provider init on web (#919)
open-feature/js-sdk: packages/client/README.md

Domains can be defined on a provider during registration.
For more details, please refer to the [providers](#providers) section.

#### Manage evaluation context for domains

By default, domain-scoped clients use the global context.

> This can be overridden by explicitly setting context when registering the provider or by references the domain when updating context.

Reviewer: ```suggestion
This can be overridden by explicitly setting context when registering the provider or by references the domain when updating context:
```

```ts

> OpenFeature.setProvider("my-domain", new NewCachedProvider(), { isCache: true });

Reviewer 2: Could we also set the targeting key here? I think this makes the example more obvious.
`isCache: true` can be confused for options or so.

Reviewer: Ya, I agree. I think something more personal like "`email`" might be clearer.

```

> Change context after the provider has been registered by using `setContext` with a name.

Reviewer: ```suggestion
To change context after the provider has been registered, use `setContext` with a name:
```
